octodns_constellix/__init__.py

```diff
@@ -311,17 +365,33 @@ def __init__(self):
 class SonarClient(object):
     BASE = 'https://api.sonar.constellix.com/rest/api'
 
+    _OCTODNS_TO_SONAR_REGION = {
```

Comment:
@ross @viranch This is for a future feature that will support automatic mapping of the pool's configured geo-setup to the monitor's region.